src/BuildScriptGenerator/PlatformInstallerBase.cs

```diff
@@ -53,7 +53,10 @@ public static void InstallPythonToolingAndLanguage(StringBuilder stringBuilder)
                 "swig",
                 "tk-dev",
                 "unixodbc-dev",
-                "uuid-dev");
+                "uuid-dev",
+                "libffi-dev",
+                "python3-dev",
+                "python3-cffi");
```

Comment:
don't we also want to dynamically install the same packages when we build python on the CLI image?